pyproject.toml

```diff
@@ -13,6 +13,12 @@ requires = [
     # meson-python if those tools are not already present on the system. No
     # need to worry about those unless one does a non-isolated build - in that
     # case they must already be installed on the system.
+    #
+    # XXX use 64-bit openblas on 64-bit windows, how to specify that?
+    'scipy-openblas32; sys_platform == "win32"',
+    # XXX macos arm64 will use accelerate, how to specify that?
+    'scipy-openblas32; sys_platform == "darwin"',
+    'scipy-openblas64; sys_platform == "linux"',
```

Comment:
We cannot add the dependency here, since it's only required for our own wheel builds. Instead it should be managed in the wheel build scripts. The changes further down in `[tool.cibuildwheel]` section do seem about right to me. And the install commands are already added to `cibw_before_build.sh`, so probably all that is needed is to remove the above 5 lines.

Reply:
We need a runtime dependency on a scipy-openblas package. How do we add the runtime dependency dynamically?

Reply:
Amend the `dependencies =` entry in `pyproject.toml` in `cibw_before_build.sh`?

Reply:
I was looking for a cleaner way, but I guess that will do.